apps/amo/tests/test_views.py

```diff
@@ -390,3 +390,13 @@ def test_contribute_json(self):
         res = self.client.get('/contribute.json')
         eq_(res.status_code, 200)
         eq_(res._headers['content-type'], ('Content-Type', 'application/json'))
+
+
+class TestRobots(amo.tests.TestCase):
+
+    def test_disable_collections(self):
+        """Make sure /en-US/firefox/collections/ gets disabled"""
+        url = reverse('collections.list')
+        response = self.client.get('/robots.txt')
+        eq_(response.status_code, 200)
```

Comment:
we use pytest `assert`s, so that would be `assert response.status_code == 200` instead